content/300-guides/300-upgrade-guides/800-upgrade-from-prisma-1/01-how-to-upgrade.mdx

```diff
@@ -38,7 +38,7 @@ On a high-level, the biggest differences between Prisma 1 and Prisma 2 are the f
   - Prisma Migrate: Data modeling and migrations (formerly `prisma deploy`).
 - Prisma 1 datamodel and `prisma.yml` have been merged into the [Prisma schema](/concepts/components/prisma-schema).
 - Prisma 2 uses its own [modeling language](https://github.com/prisma/specs/tree/master/schema) instead of being based on GraphQL SDL.
-- Prisma 2 doesn't expose ["a GraphQL API for your database"](https://www.prisma.io/blog/prisma-and-graphql-mfl5y2r7t49c/) anymore, but only allows for _programmatic access_ via the Prisma Client API. This means Prisma 2 [doesn't support Prisma binding](https://www.prisma.io/docs/more/faq#does-prisma-client-support-graphql-schema-delegation-and-graphql-binding) any more.
+- Prisma 2 doesn't expose ["a GraphQL API for your database"](https://www.prisma.io/blog/prisma-and-graphql-mfl5y2r7t49c/) anymore, but only allows for _programmatic access_ via the Prisma Client API. This means Prisma 2 [doesn't support Prisma binding](/about/prisma/faq#does-prisma-client-support-graphql-schema-delegation-and-graphql-binding) any more.
```

Comment:
Using the external URL also hid a link that was linking to an old URL that we are now redirecting somewhere else. 
Using the internal URL the link checker found this and failed with "broken link".